packages/paste-website/src/pages/components/button/index.mdx

```diff
@@ -117,61 +117,51 @@ screen readers and dictation software. Here are some quick tips:
 ### Examples
 
 <LivePreview scope={{Button, Box}} language="jsx">
-  {`<div>
+  {`<Box>
```

Comment:
If you're going to the effort of removing the div, should you convert to use stack on these?